frontend/src/components/admin/UsersTable.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { Icon } from "@chakra-ui/icon";
+import { MdDelete } from "react-icons/md";
+import {
+  Badge,
+  Link,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  IconButton,
+} from "@chakra-ui/react";
+import { User } from "../../APIClients/queries/UserQueries";
+import convertLanguageTitleCase from "../../utils/LanguageUtils";
+
+export type UsersTableProps = {
+  isTranslators?: boolean;
+  users: User[];
+};
+
+const UsersTable = ({ isTranslators = false, users }: UsersTableProps) => {
+  const approvedLanguanges = users.map((userObj: User) => {
+    const approvedLanguange = isTranslators
+      ? userObj?.approvedLanguagesTranslation
+      : userObj?.approvedLanguagesReview;
+    return approvedLanguange
+      ? JSON.parse(approvedLanguange!!.trim().replace(/'/g, '"'))
+      : "";
+  });
+
+  const generateBadges = (approvedLanguages: any) =>
+    Object.keys(approvedLanguages).map((apprLang: string) => (
+      // TODO: determine badge color by label
+      <Badge background="orange.50">{`${convertLanguageTitleCase(
+        apprLang,
+      )} | Level ${approvedLanguages[apprLang]}`}</Badge>
+    ));
+
+  const tableBody = users.map((userObj: User, index: number) => (
+    <Tr>
+      <Td>
+        <Link
```

Comment:
the user/email links don't seem to work for me:
![links](https://user-images.githubusercontent.com/32009013/137340014-f734658f-f4fc-46e9-a6a8-5a194ac47bbf.gif)

Reply:
AHHHH IM A DUMB CUCUMBER AND FORGOT TO PUSH UP CHANGES LOL